<?xml version="1.0" encoding="UTF-8"?>
<testsuite xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://maven.apache.org/surefire/maven-surefire-plugin/xsd/surefire-test-report-3.0.xsd" version="3.0" name="org.example.ACTest" time="0.028" tests="4" errors="0" skipped="0" failures="4">
  <properties>
    <property name="java.specification.version" value="20"/>
    <property name="sun.cpu.isalist" value="amd64"/>
    <property name="sun.jnu.encoding" value="Cp1251"/>
    <property name="java.class.path" value="E:\Univ\ООП\OOP-Lab1-Pysarenkov\target\test-classes;E:\Univ\ООП\OOP-Lab1-Pysarenkov\target\classes;C:\Users\Tymofii\.m2\repository\junit\junit\3.8.1\junit-3.8.1.jar;"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="user.variant" value=""/>
    <property name="java.vendor.url" value="https://java.oracle.com/"/>
    <property name="user.timezone" value="Europe/Kiev"/>
    <property name="os.name" value="Windows 11"/>
    <property name="java.vm.specification.version" value="20"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="user.country" value="UA"/>
    <property name="sun.boot.library.path" value="C:\Users\Tymofii\.jdks\openjdk-20.0.2\bin"/>
    <property name="sun.java.command" value="C:\Users\Tymofii\AppData\Local\Temp\surefire9347127276132191686\surefirebooter-20231023205610013_3.jar C:\Users\Tymofii\AppData\Local\Temp\surefire9347127276132191686 2023-10-23T20-56-09_924-jvmRun1 surefire-20231023205610013_1tmp surefire_0-20231023205610013_2tmp"/>
    <property name="jdk.debug" value="release"/>
    <property name="surefire.test.class.path" value="E:\Univ\ООП\OOP-Lab1-Pysarenkov\target\test-classes;E:\Univ\ООП\OOP-Lab1-Pysarenkov\target\classes;C:\Users\Tymofii\.m2\repository\junit\junit\3.8.1\junit-3.8.1.jar;"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="user.home" value="C:\Users\Tymofii"/>
    <property name="user.language" value="uk"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="java.version.date" value="2023-07-18"/>
    <property name="java.home" value="C:\Users\Tymofii\.jdks\openjdk-20.0.2"/>
    <property name="file.separator" value="\"/>
    <property name="basedir" value="E:\Univ\ООП\OOP-Lab1-Pysarenkov"/>
    <property name="java.vm.compressedOopsMode" value="Zero based"/>
    <property name="line.separator" value="&#10;"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="surefire.real.class.path" value="C:\Users\Tymofii\AppData\Local\Temp\surefire9347127276132191686\surefirebooter-20231023205610013_3.jar"/>
    <property name="user.script" value=""/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="java.runtime.version" value="20.0.2+9-78"/>
    <property name="user.name" value="Tymofii"/>
    <property name="stdout.encoding" value="Cp1251"/>
    <property name="path.separator" value=";"/>
    <property name="os.version" value="10.0"/>
    <property name="java.runtime.name" value="OpenJDK Runtime Environment"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="java.vm.name" value="OpenJDK 64-Bit Server VM"/>
    <property name="localRepository" value="C:\Users\Tymofii\.m2\repository"/>
    <property name="java.vendor.url.bug" value="https://bugreport.java.com/bugreport/"/>
    <property name="java.io.tmpdir" value="C:\Users\Tymofii\AppData\Local\Temp\"/>
    <property name="idea.version" value="2023.2.1"/>
    <property name="java.version" value="20.0.2"/>
    <property name="user.dir" value="E:\Univ\ООП\OOP-Lab1-Pysarenkov"/>
    <property name="os.arch" value="amd64"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="sun.os.patch.level" value=""/>
    <property name="native.encoding" value="Cp1251"/>
    <property name="java.library.path" value="C:\Users\Tymofii\.jdks\openjdk-20.0.2\bin;C:\Windows\Sun\Java\bin;C:\Windows\system32;C:\Windows;E:\Ruby32-x64\bin;C:\Program Files\Common Files\Oracle\Java\javapath;c:\windows\system32;c:\windows;c:\windows\system32\wbem;c:\windows\system32\windowspowershell\v1.0\;c:\windows\system32\openssh\;c:\program files\dotnet\;c:\program files\nvidia corporation\nvidia nvdlisr;E:\Program Files\Microsoft VS Code\bin;E:\Program Files\Go\bin;C:\Program Files\Git\cmd;E:\Program Files\python\Scripts\;E:\Program Files\python\;C:\Users\Tymofii\AppData\Local\Microsoft\WindowsApps;E:\Program Files\IntelliJ IDEA 2023.2.1\bin;;C:\Users\Tymofii\go\bin;C:\Users\Tymofii\AppData\Local\GitHubDesktop\bin;E:\Program Files\JDK\bin;E:\msys64\ucrt64\bin;;E:\Program Files\RubyMine 2023.2.3\bin;;."/>
    <property name="java.vm.info" value="mixed mode, sharing"/>
    <property name="stderr.encoding" value="Cp1251"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="java.vm.version" value="20.0.2+9-78"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="java.class.version" value="64.0"/>
  </properties>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.004">
    <failure message="Exception in constructor: testTotalSeats (java.lang.RuntimeException: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db&#10;	at org.example.ACTest.&lt;init&gt;(ACTest.java:17)&#10;	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)&#10;	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)&#10;	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:135)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:114)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:75)&#10;	at org.example.ACTest.suite(ACTest.java:22)&#10;	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#10;	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&#10;	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.createInstanceFromSuiteMethod(JUnit3Reflector.java:133)&#10;	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.constructTestObject(JUnit3Reflector.java:108)&#10;	at org.apache.maven.surefire.junit.JUnitTestSetExecutor.execute(JUnitTestSetExecutor.java:59)&#10;	at org.apache.maven.surefire.junit.JUnit3Provider.executeTestSet(JUnit3Provider.java:131)&#10;	at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:93)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)&#10;Caused by: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db&#10;	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:708)&#10;	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)&#10;	at org.example.AirComp.parsePlanes(AirComp.java:39)&#10;	at org.example.AirComp.&lt;init&gt;(AirComp.java:20)&#10;	at org.example.AirComp.getAC(AirComp.java:32)&#10;	at org.example.ACTest.&lt;init&gt;(ACTest.java:15)&#10;	... 18 more&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: 
Exception in constructor: testTotalSeats (java.lang.RuntimeException: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db
	at org.example.ACTest.<init>(ACTest.java:17)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
	at junit.framework.TestSuite.createTest(TestSuite.java:135)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:114)
	at junit.framework.TestSuite.<init>(TestSuite.java:75)
	at org.example.ACTest.suite(ACTest.java:22)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.createInstanceFromSuiteMethod(JUnit3Reflector.java:133)
	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.constructTestObject(JUnit3Reflector.java:108)
	at org.apache.maven.surefire.junit.JUnitTestSetExecutor.execute(JUnitTestSetExecutor.java:59)
	at org.apache.maven.surefire.junit.JUnit3Provider.executeTestSet(JUnit3Provider.java:131)
	at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:93)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
Caused by: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:708)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)
	at org.example.AirComp.parsePlanes(AirComp.java:39)
	at org.example.AirComp.<init>(AirComp.java:20)
	at org.example.AirComp.getAC(AirComp.java:32)
	at org.example.ACTest.<init>(ACTest.java:15)
	... 18 more
)
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:263)
	at junit.framework.TestCase.runBare(TestCase.java:127)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.apache.maven.surefire.junit.JUnitTestSetExecutor.execute(JUnitTestSetExecutor.java:81)
	at org.apache.maven.surefire.junit.JUnit3Provider.executeTestSet(JUnit3Provider.java:131)
	at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:93)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0">
    <failure message="Exception in constructor: testTotalCapacity (java.lang.RuntimeException: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db&#10;	at org.example.ACTest.&lt;init&gt;(ACTest.java:17)&#10;	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)&#10;	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)&#10;	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:135)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:114)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:75)&#10;	at org.example.ACTest.suite(ACTest.java:22)&#10;	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#10;	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&#10;	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.createInstanceFromSuiteMethod(JUnit3Reflector.java:133)&#10;	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.constructTestObject(JUnit3Reflector.java:108)&#10;	at org.apache.maven.surefire.junit.JUnitTestSetExecutor.execute(JUnitTestSetExecutor.java:59)&#10;	at org.apache.maven.surefire.junit.JUnit3Provider.executeTestSet(JUnit3Provider.java:131)&#10;	at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:93)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)&#10;Caused by: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db&#10;	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:708)&#10;	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)&#10;	at org.example.AirComp.parsePlanes(AirComp.java:39)&#10;	at org.example.AirComp.&lt;init&gt;(AirComp.java:20)&#10;	at org.example.AirComp.getAC(AirComp.java:32)&#10;	at org.example.ACTest.&lt;init&gt;(ACTest.java:15)&#10;	... 18 more&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: 
Exception in constructor: testTotalCapacity (java.lang.RuntimeException: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db
	at org.example.ACTest.<init>(ACTest.java:17)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
	at junit.framework.TestSuite.createTest(TestSuite.java:135)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:114)
	at junit.framework.TestSuite.<init>(TestSuite.java:75)
	at org.example.ACTest.suite(ACTest.java:22)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.createInstanceFromSuiteMethod(JUnit3Reflector.java:133)
	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.constructTestObject(JUnit3Reflector.java:108)
	at org.apache.maven.surefire.junit.JUnitTestSetExecutor.execute(JUnitTestSetExecutor.java:59)
	at org.apache.maven.surefire.junit.JUnit3Provider.executeTestSet(JUnit3Provider.java:131)
	at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:93)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
Caused by: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:708)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)
	at org.example.AirComp.parsePlanes(AirComp.java:39)
	at org.example.AirComp.<init>(AirComp.java:20)
	at org.example.AirComp.getAC(AirComp.java:32)
	at org.example.ACTest.<init>(ACTest.java:15)
	... 18 more
)
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:263)
	at junit.framework.TestCase.runBare(TestCase.java:127)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.apache.maven.surefire.junit.JUnitTestSetExecutor.execute(JUnitTestSetExecutor.java:81)
	at org.apache.maven.surefire.junit.JUnit3Provider.executeTestSet(JUnit3Provider.java:131)
	at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:93)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.001">
    <failure message="Exception in constructor: testSortByRange (java.lang.RuntimeException: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db&#10;	at org.example.ACTest.&lt;init&gt;(ACTest.java:17)&#10;	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)&#10;	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)&#10;	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:135)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:114)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:75)&#10;	at org.example.ACTest.suite(ACTest.java:22)&#10;	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#10;	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&#10;	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.createInstanceFromSuiteMethod(JUnit3Reflector.java:133)&#10;	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.constructTestObject(JUnit3Reflector.java:108)&#10;	at org.apache.maven.surefire.junit.JUnitTestSetExecutor.execute(JUnitTestSetExecutor.java:59)&#10;	at org.apache.maven.surefire.junit.JUnit3Provider.executeTestSet(JUnit3Provider.java:131)&#10;	at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:93)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)&#10;Caused by: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db&#10;	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:708)&#10;	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)&#10;	at org.example.AirComp.parsePlanes(AirComp.java:39)&#10;	at org.example.AirComp.&lt;init&gt;(AirComp.java:20)&#10;	at org.example.AirComp.getAC(AirComp.java:32)&#10;	at org.example.ACTest.&lt;init&gt;(ACTest.java:15)&#10;	... 18 more&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: 
Exception in constructor: testSortByRange (java.lang.RuntimeException: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db
	at org.example.ACTest.<init>(ACTest.java:17)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
	at junit.framework.TestSuite.createTest(TestSuite.java:135)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:114)
	at junit.framework.TestSuite.<init>(TestSuite.java:75)
	at org.example.ACTest.suite(ACTest.java:22)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.createInstanceFromSuiteMethod(JUnit3Reflector.java:133)
	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.constructTestObject(JUnit3Reflector.java:108)
	at org.apache.maven.surefire.junit.JUnitTestSetExecutor.execute(JUnitTestSetExecutor.java:59)
	at org.apache.maven.surefire.junit.JUnit3Provider.executeTestSet(JUnit3Provider.java:131)
	at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:93)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
Caused by: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:708)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)
	at org.example.AirComp.parsePlanes(AirComp.java:39)
	at org.example.AirComp.<init>(AirComp.java:20)
	at org.example.AirComp.getAC(AirComp.java:32)
	at org.example.ACTest.<init>(ACTest.java:15)
	... 18 more
)
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:263)
	at junit.framework.TestCase.runBare(TestCase.java:127)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.apache.maven.surefire.junit.JUnitTestSetExecutor.execute(JUnitTestSetExecutor.java:81)
	at org.apache.maven.surefire.junit.JUnit3Provider.executeTestSet(JUnit3Provider.java:131)
	at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:93)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]></failure>
  </testcase>
  <testcase name="warning" classname="junit.framework.TestSuite$1" time="0.001">
    <failure message="Exception in constructor: testFilter (java.lang.RuntimeException: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db&#10;	at org.example.ACTest.&lt;init&gt;(ACTest.java:17)&#10;	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)&#10;	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)&#10;	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)&#10;	at junit.framework.TestSuite.createTest(TestSuite.java:135)&#10;	at junit.framework.TestSuite.addTestMethod(TestSuite.java:114)&#10;	at junit.framework.TestSuite.&lt;init&gt;(TestSuite.java:75)&#10;	at org.example.ACTest.suite(ACTest.java:22)&#10;	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)&#10;	at java.base/java.lang.reflect.Method.invoke(Method.java:578)&#10;	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.createInstanceFromSuiteMethod(JUnit3Reflector.java:133)&#10;	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.constructTestObject(JUnit3Reflector.java:108)&#10;	at org.apache.maven.surefire.junit.JUnitTestSetExecutor.execute(JUnitTestSetExecutor.java:59)&#10;	at org.apache.maven.surefire.junit.JUnit3Provider.executeTestSet(JUnit3Provider.java:131)&#10;	at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:93)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)&#10;	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)&#10;Caused by: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db&#10;	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:708)&#10;	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)&#10;	at org.example.AirComp.parsePlanes(AirComp.java:39)&#10;	at org.example.AirComp.&lt;init&gt;(AirComp.java:20)&#10;	at org.example.AirComp.getAC(AirComp.java:32)&#10;	at org.example.ACTest.&lt;init&gt;(ACTest.java:15)&#10;	... 18 more&#10;)" type="junit.framework.AssertionFailedError"><![CDATA[junit.framework.AssertionFailedError: 
Exception in constructor: testFilter (java.lang.RuntimeException: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db
	at org.example.ACTest.<init>(ACTest.java:17)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
	at junit.framework.TestSuite.createTest(TestSuite.java:135)
	at junit.framework.TestSuite.addTestMethod(TestSuite.java:114)
	at junit.framework.TestSuite.<init>(TestSuite.java:75)
	at org.example.ACTest.suite(ACTest.java:22)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.createInstanceFromSuiteMethod(JUnit3Reflector.java:133)
	at org.apache.maven.surefire.common.junit3.JUnit3Reflector.constructTestObject(JUnit3Reflector.java:108)
	at org.apache.maven.surefire.junit.JUnitTestSetExecutor.execute(JUnitTestSetExecutor.java:59)
	at org.apache.maven.surefire.junit.JUnit3Provider.executeTestSet(JUnit3Provider.java:131)
	at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:93)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
Caused by: java.sql.SQLException: No suitable driver found for jdbc:sqlite:aviakomp.db
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:708)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:253)
	at org.example.AirComp.parsePlanes(AirComp.java:39)
	at org.example.AirComp.<init>(AirComp.java:20)
	at org.example.AirComp.getAC(AirComp.java:32)
	at org.example.ACTest.<init>(ACTest.java:15)
	... 18 more
)
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.TestSuite$1.runTest(TestSuite.java:263)
	at junit.framework.TestCase.runBare(TestCase.java:127)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.apache.maven.surefire.junit.JUnitTestSetExecutor.execute(JUnitTestSetExecutor.java:81)
	at org.apache.maven.surefire.junit.JUnit3Provider.executeTestSet(JUnit3Provider.java:131)
	at org.apache.maven.surefire.junit.JUnit3Provider.invoke(JUnit3Provider.java:93)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]></failure>
  </testcase>
</testsuite>